packages/app/src/components/home/HomePage.tsx

```diff
@@ -100,6 +101,9 @@ export const HomePage = () => (
             ]}
           />
         </Grid>
+        <Grid item xs={12} md={4}>
+          <CalendarCard />
```

Comment:
Do you imagine this component being used in other places of Backstage too? If not - I think it would be nice prefixing with HomePage* as the other components. If you think it will be used in other parts of backstage - feel free to keep as it is!